examples/graphene_apps/python_worker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:bionic as base_image
```

Comment:
Where is the Dockerfile used?

Reply:
This docker file is used by `examples/graphene_apps/python_worker/docker-compose.yaml`